homeassistant/components/logger.py

```diff
@@ -43,6 +47,11 @@
 }, extra=vol.ALLOW_EXTRA)
 
 
+def set_default_level(hass, level):
```

Comment:
No need to add a function. Won't be used by other components.

Reply:
If I remove the function, will `python_script` still be able to access the feature, perhaps by calling the service instead?